mods/essb/abilities.js

```diff
@@ -554,25 +554,23 @@ exports.BattleAbilities = {
 			move.stab = 2;
 		},
 	},
-	"cartoonphysics": {
-		id: "cartoonphysics",
-		name: "Cartoon Physics",
-		onStart: function (pokemon) {
-			this.add('-ability', pokemon, 'Cartoon Physics');
-		},
-		onModifyMove: function (move) {
-			move.ignoreAbility = true;
-		},
-		onSourceModifyDamage: function (damage, source, target, move) {
-			if (this.random(10) < 5) {
-				return this.chainModify(false);
+	"sysop": {
+		id: "sysop",
+		name: "Sys-OP",
+		onPrepareHit: function (source, target, move) {
```

Comment:
If this has Wonder Guard AND Protean it'll keep changing type and players won't be able to KO it....